vxquery-core/src/test/java/org/apache/vxquery/datamodel/AbstractPointableTest.java

```diff
@@ -28,7 +28,9 @@
 
     protected void getTaggedValuePointable(Object value, IPointable result) throws IOException {
```

Comment:
And the getKeys method of ObjectPointable: should it return a sequence pointable of keys which are string pointables without valuetag?